doc/api/fs.md

```diff
@@ -6914,6 +6917,7 @@ their synchronous counterparts are of this type.
 If `bigint` in the `options` passed to those methods is true, the numeric values
 will be `bigint` instead of `number`, and the object will contain additional
 nanosecond-precision properties suffixed with `Ns`.
+`Stat` objects are not to be created directly using the `new` keyword.
```

Comment:
It would be nice to add note about instance type as well.
Unless we add `fs.isStats()`, something among the lines "Prefer duck typing check over `X instanceof fs.Stats`, e.g. `typeof X.size === 'bigint' && typeof X.blocks === 'bigint' && ...`".